packages/scripts/README.md

```diff
@@ -65,22 +66,31 @@ A preset is a node module `talend-scripts-preset-${presetName}`. It exports 3 fu
 
 ```javascript
 module.exports = {
-	getWebpackConfiguration(presetApi) {},
-	getJestConfiguration(presetApi) {}, // TODO
-	getEslintConfiguration(presetApi) {} // TODO
+	getWebpackConfiguration(presetApi) { },
+	getEslintConfiguration(presetApi) { },
 }
 ```
+| Preset function | Description |
+|---|---|
+| getWebpackConfiguration | Returns the webpack configuration object. |
+| getEslintConfiguration | Returns the path to .eslintrc file. It will be passed to eslint --config option. This path should be absolute or relative from cwd. |
+
 
 The preset api contains the run mode and utility functions
 
 ```javascript
-const { mode, getUserConfig } = presetApi;
+const {
+	mode,
+	getUserConfig,
+	hereRelative,
+} = presetApi;
 ```
 
 | Preset api | Description |
 |---|---|
 | mode | The run mode : `development` or `production` |
 | getUserConfig(path: array or string, defaultValue: any) | User configuration getter. They are defined in the `talend-scrips.json` file. It uses lodash.get under the hood, the object path can be an array of keys of a dot-separated string. |
+| hereRelative(dirname: string, relativePathFromDirname: string) | Resolve a relative path from cwd, based on the current folder and the file relative path |
```

Comment:
here relative? Do you want a `getRelativePath`as helper?

Reply:
It's to have path from cwd. But in fact you're right, it's not usefull, we can request an absolute path from `getEslintConfigurationPath` to pass to eslint cli.
Let's remove that